core/templates/components/forms/schema-based-editors/schema-based-float-editor.component.ts

```diff
@@ -185,7 +185,7 @@ implements ControlValueAccessor, OnInit, Validator {
       this.numericInputValidationService.validateNumber(
         this.localValue,
         this.checkRequireNonnegativeInputValue,
-        this.getCurrentDecimalSeparator()));
+        this.getCurrentDecimalSeparator())) || '';
```

Comment:
Why are we defaulting to '' rather than null, here and below?

Isn't null more semantically correct (because '' isn't a valid i18n key)?

Reply:
Done.